Detectors/CTP/workflow/src/RawDecoderSpec.cxx

```diff
@@ -111,17 +111,25 @@ void RawDecoderSpec::run(framework::ProcessingContext& ctx)
   //
   std::vector<LumiInfo> lumiPointsHBF1;
   std::vector<InputSpec> filter{InputSpec{"filter", ConcreteDataTypeMatcher{"CTP", "RAWDATA"}, Lifetime::Timeframe}};
+  {
+    int payloadSize = 0;
+    for (const auto& ref : o2::framework::InputRecordWalker(inputs, filter)) {
+      const auto dh = o2::framework::DataRefUtils::getHeader<o2::header::DataHeader*>(ref);
+      payloadSize += o2::framework::DataRefUtils::getPayloadSize(ref);
+    }
+    if ((mMaxInputSize > 0) && (payloadSize > mMaxInputSize)) {
```

Comment:
@lietava looks fine, but what is the max possible size in case of black data? It might be safer to accumulate the size in the `size_t` instead of `int` and cast the `mMaxInputSize` to size_t before comparison.

Reply:
Size of the data is 
10 khz        10112
0.54 MHz   45000
1.47 MHz 101000
so I guess 40MHz is ~ 2.7e6  but I want to measure with data at p2.
(I will do size_t)

Reply:
size_t done